src/spider/worker/worker.cpp

```
@@ -106,7 +109,20 @@ auto heartbeat_loop(
     while (!stop_token.stop_requested()) {
         std::this_thread::sleep_for(std::chrono::seconds(1));
         spdlog::debug("Updating heartbeat");
-        spider::core::StorageErr const err = metadata_store->update_heartbeat(driver.get_id());
+        std::variant<spider::core::MySqlConnection, spider::core::StorageErr> conn_result
+                = spider::core::MySqlConnection::create(metadata_store->get_url());
+        if (std::holds_alternative<spider::core::StorageErr>(conn_result)) {
+            spdlog::error(
+                    "Failed to connection to storage: {}",
```

Cppcheck (2.10-2)

[performance] Variable 'm_id' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)